include/MainApplication.h

```diff
@@ -34,6 +34,11 @@
 #include <QAbstractNativeEventFilter>
 #endif
 
+
+namespace lmms
+{
+
+
 #if defined(LMMS_BUILD_WIN32)
 class MainApplication : public QApplication, public QAbstractNativeEventFilter
```

Comment:
Perhaps this should be in `lmms::gui`? `QApplication` is specifically for QWidget-based applications; the core-only version is `QCoreApplication`. Also, this class calls into a GUI class as part of handling the opening of files, and on Windows, it does some handling of native window messages.

Reply:
Implemented by @allejok96